tsconfig.json

```diff
@@ -6,10 +6,12 @@
     }
   },
   "compilerOptions": {
+    "emitDeclarationOnly": true,
```

Comment:
I'm using babel instead of tsc to generate js files to be able to use the transform runtime plugin
https://www.typescriptlang.org/docs/handbook/babel-with-typescript.html